app/controllers/completed_applications_controller.rb

```diff
@@ -30,12 +30,7 @@ def applications_from_datastore
       status: status_filter, **pagination_params
     ).call
 
-    @paginator = InfinitePaginationV2.new(
-      pagination: result.pagination,
-      params: params,
-    )
-
-    result
+    Kaminari.paginate_array(result, total_count: result.pagination['total_count']).page(params[:page])
```

Comment:
Nice!
Tiny thing but can we have the `.page(params[:page])` up in the `#index` action, as to unify the interface in both controllers. No blocker!